store/tikv/oracle/oracles/pd.go

```diff
@@ -231,3 +238,47 @@ func (o *pdOracle) GetLowResolutionTimestampAsync(ctx context.Context, opt *orac
 		err: nil,
 	}
 }
+
+func (o *pdOracle) getStaleTimestamp(ctx context.Context, prevSecond int64) (uint64, error) {
+	ts, ok := o.getLastTS(oracle.GlobalTxnScope)
+	if !ok {
+		return 0, errors.Errorf("get stale timestamp, invalid txnScope = %s", oracle.GlobalTxnScope)
+	}
+	tsArrival, ok := o.getLastArrivalTS()
+	if !ok {
+		return 0, errors.Errorf("get last arrival timestamp, invalid txnScope = %s", oracle.GlobalTxnScope)
+	}
+	arrivalTime := oracle.GetTimeFromTS(tsArrival)
+	physicalTime := oracle.GetTimeFromTS(ts)
+	if physicalTime.Unix() <= prevSecond {
+		return 0, errors.Errorf("get invalid prevSecond, prevSecond must less than physicalTime, "+
+			"get prevSecond: %d, physicalTime: %d", prevSecond, physicalTime.Unix())
```

Comment:
```suggestion
		return 0, errors.Errorf("invalid prevSecond %v", prevSecond)
```

Reply:
Thanks for your review, I agree with you.